pkg/scheduler/framework/plugins/noderesources/fit_test.go

```diff
@@ -732,13 +732,16 @@ func TestRestartableInitContainers(t *testing.T) {
 			}
 			cycleState := framework.NewCycleState()
 			_, preFilterStatus := p.(framework.PreFilterPlugin).PreFilter(context.Background(), cycleState, test.pod)
+			if diff := cmp.Diff(preFilterStatus, test.wantStatus); diff != "" {
```

Comment:
Maybe we should add both `wantFilterStatus` and `wantPreFilterStatus` and compare them both.

Reply:
done